packages/integrations/src/download-client/aria2/aria2-client.ts

```diff
@@ -0,0 +1,73 @@
+
+import { aria2, Conn, ReadyState, open, PreconfiguredSocket, OpenOptions } from "maria2/dist"
```

Comment:
Check lint errors. For future reference, please fix those before you request a review.